cg/meta/workflow/mutant.py

```diff
@@ -55,7 +55,12 @@ def get_trailblazer_config_path(self, case_id: str) -> Path:
         return Path(self.get_case_output_path(case_id=case_id), "trailblazer_config.yaml")
 
     def get_sample_fastq_destination_dir(self, case_obj: models.Family, sample_obj: models.Sample):
```

Comment:
```suggestion
    def get_sample_fastq_destination_dir(self, case_id: str, sample_obj: models.Sample) -> Path:
```
Since we only use the case for its id we can maybe just pass that?